cnames_active.js

```diff
@@ -443,6 +443,7 @@ var cnames_active = {
   "city": "city.github.io",
   "civet": "civet-org.github.io",
   "cjss": "cjss-group.github.io/CJSS",
+  "ck": "chunkeat99.github.io",
```

Comment:
```suggestion
  "ck": "chunkeat99.github.io/ck.js.org",
```
I'm guessing you want to target https://github.com/chunkeat99/ck.js.org, so this needs to be updated